nativelib/src/main/scala/scala/scalanative/native/time.scala

```diff
@@ -0,0 +1,27 @@
+package scala.scalanative
+package native
+
+@extern
+object time {
+
+  type time_t = CLong
+
+  @struct
+  class timespec(
+      val tv_sec: time_t = 0L, /* seconds */
+      val tv_nsec: CLong = 0L /* nanoseconds */
+  )
+
+  def clock_getres(clk_id: Clock.clockid_t, res: Ptr[timespec]): CInt  = extern
+  def clock_gettime(clk_id: Clock.clockid_t, res: Ptr[timespec]): CInt = extern
+  def clock_settime(clk_id: Clock.clockid_t, res: Ptr[timespec]): CInt = extern
```

Comment:
Couldn't we tag with a specific comment the related parts of code?
In this way when we will have #58 merged we can move the code to the proper subproject.

Another option is to start to have a couple of subprojects like ```posix-nativelib``` and ```windows-nativelib``` that can be switched in user build imports but that respect the very same classes/methods signatures.

Probably people implementing things on unix and win environment are potentially different (i.e. I have no win dev machine to test on).